packages/sdk/src/swap/sdk.ts

```diff
@@ -238,7 +238,7 @@ export class SwapSDK {
   getStatus(
     swapStatusRequest: SwapStatusRequest,
     options: ApiService.RequestOptions = {},
-  ): Promise<SwapStatusResponse> {
+  ): Promise<SwapV2.SwapStatusResponse> {
```

Comment:
i think we need to do a separate `getStatusV2` method and keep the behaviour of this one for backwards compatibility